### ISSUE TITLE
The exporter live preview does not take into account the active status of entity, 0.8

### ISSUE THREAD

**Issue by ArttuTupala** [spine-tools/Spine-Toolbox#2612]
The live preview shows the not active members, but running the exporter correctly does not include them. 
Confuses the user to think that the problem is with the exporter when the entities are actually just not active

**Comment by soininen** [comment]
Currently, Exporter does not know which scenario the live preview should show, especially when there is more than one scenario selected. We could add a combo box in the Specification editor where users could select a scenario to preview, though.

**Comment by manuelma** [comment]
Yes, I like that solution. We could even reuse the 'funnel' button in DB Editor toolbar.
